
package logica.servidor;

import jakarta.xml.ws.WebFault;


/**
 * This class was generated by the XML-WS Tools.
 * XML-WS Tools 4.0.0
 * Generated source version: 3.0
 * 
 */
@WebFault(name = "ExceptionUsuarioNickRepetido",  targetNamespace = "http://servidor.logica/")
public class ExceptionUsuarioNickRepetido_Exception
    extends Exception
{

    /**
     * Java type that goes as soapenv:Fault detail element.
     * 
     */
    private ExceptionUsuarioNickRepetido faultInfo;

    /**
     * 
     * @param faultInfo
     * @param message
     */
    public ExceptionUsuarioNickRepetido_Exception(String message,  ExceptionUsuarioNickRepetido faultInfo) {
        super(message);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @param cause
     * @param faultInfo
     * @param message
     */
    public ExceptionUsuarioNickRepetido_Exception(String message,  ExceptionUsuarioNickRepetido faultInfo,  Throwable cause) {
        super(message,  cause);
        this.faultInfo = faultInfo;
    }

    /**
     * 
     * @return
     *     returns fault bean: logica.servidor.ExceptionUsuarioNickRepetido
     */
    public ExceptionUsuarioNickRepetido getFaultInfo() {
        return faultInfo;
    }

}
